pkg/reexec/command_solaris.go

```diff
@@ -0,0 +1,23 @@
+// +build solaris
+
+package reexec
+
+import (
+	"os/exec"
+)
+
+// Self returns the path to the current process's binary.
```

Comment:
Same as above, it's the same as `command_freebsd.go`, maybe it would be better to have a `command_unix.go` instead.